app/assets/stylesheets/nav.scss

```diff
@@ -15,10 +15,10 @@
   height: $topbar-height;
 }
 
-.droplink li {
-  background: transparent;
+.top-bar-section ul li a {
```

Comment:
Merge rule `.top-bar-section ul li a` with rule on line 7<br>Selector should have depth of applicability no greater than 2, but was 4